code/obj/item/grenades.dm

```diff
@@ -478,7 +478,7 @@ PIPE BOMBS + CONSTRUCTION
 					var/area/t = get_area(O)
 					if(t && t.sanctuary) continue
 					O.emp_act()
-
+					SEND_SIGNAL(O, COMSIG_MOVABLE_EMP_ACT)
```

Comment:
The EMP thing I mentioned above applies here too if you want to change it.

Reply:
Done.